svix-cli/Cargo.toml

```diff
@@ -1,12 +1,24 @@
 [package]
 name = "svix-cli"
 description = "A CLI to interact with the Svix API."
+homepage = "https://www.svix.com"
 authors = ["Svix Inc. <oss@svix.com>"]
 version = "1.44.0"
 edition = "2021"
 license = "MIT"
 keywords = ["svix", "webhooks", "diahook"]
 categories = ["development-tools", "asynchronous", "network-programming", "web-programming"]
+repository = "https://github.com/svix/svix-webhooks"
+publish = false
```

Comment:
I think we do want to publish to crates.io?

Reply:
Yeah, I think we can (and should) but I wanted to do the publishes that involve external services as a 2nd step. Let's turn this one on when enabling homebrew 🙏🏻